### DISCUSSION TITLE
Modelling a pool of revolvable assets

### DISCUSSION THREAD

**Discussion by Orangemaple23** [yellowbean/AbsBox#67] [General]
Hello @yellowbean,

Thank you for adding the invoice asset type!
 
I am trying to add more than one asset that can be bought in the revolvable assets. (Please see the jupyter notebook attached). 

However, I am unsure as to what the syntax looks like for having more than one asset as the revolvable assets for both invoices and loans/mortgages. Currently, I get an error when trying to do so. 
[revolving_asset_list.zip](https://github.com/yellowbean/AbsBox/files/15296554/revolving_asset_list.zip)

Would be grateful for any help!

**Comment by yellowbean** [comment]
### Reference
https://absbox-doc.readthedocs.io/en/latest/analytics.html#revolving-assumption

the syntax takes a list of assets:

  ["constant",asset1,asset2....]

but in the attached notebook the code is 

   ["constant", [asset1,asset2...] ]

A quick fix by adding an `asterisk`  should look as below. ( In python it will *expand* a list )

![Screenshot 2024-05-13 at 23 19 09](https://github.com/yellowbean/AbsBox/assets/1008321/f1b62436-e651-4263-a8ae-ab2b7d09a241)

**Reply by Orangemaple23** [reply]
Ahhh understood, thank you for the clarification!
